Makefile

```diff
@@ -45,7 +45,9 @@ watch: .venv
 
 check-typing: .venv
 	@echo '==> Checking types'
-	.venv/bin/mypy --strict $(SRC)
+	# redundant cast is temporarily disabled because it's not consistent
+	# feel free to enable it with future versions of mypy
+	.venv/bin/mypy --strict --no-warn-redundant-casts $(SRC)
```

Comment:
I think this is better in the `.mypy.ini` file, both the `--strict` and the `--no-warn-redundant-casts` flag.

Reply:
Eh, obviously. Fixed, thanks.